lib/operations/collection_ops.js

```diff
@@ -892,6 +892,63 @@ function insertOne(coll, doc, options, callback) {
   });
 }
 
+/**
+ * Inserts an array of documents into MongoDB. If documents passed in do not contain the **_id** field,
+ * one will be added to each of the documents missing it by the driver, mutating the document. This behavior
+ * can be overridden by setting the **forceServerObjectId** flag.
+ *
+ * @method
+ * @param {Collection} a Collection instance.
+ * @param {object[]} docs Documents to insert.
+ * @param {number} [options] Optional settings. See Collection.prototype.insertMany for a list of options.
+ * @param {Collection~insertWriteOpCallback} [callback] The command result callback
+ */
+function insertMany(coll, docs, options, callback) {
+  if (!Array.isArray(docs) && typeof callback === 'function') {
```

Comment:
since this is inside `executeOperation`, `typeof callback === 'function'` will always be true. We can get rid of the promise case.